generators/app/index.js

```diff
@@ -1,114 +1,109 @@
-var generators = require('yeoman-generator');
-var _ = require('lodash');
+var generators = require('yeoman-generator')
+var _ = require('lodash')
 
 module.exports = generators.Base.extend({
 
   _paths: function () {
-    this.log( this.destinationRoot() );
-    this.log( this.destinationPath( this.classname + '.js') );
-    this.log( this.sourceRoot() );
-    this.log( this.templatePath('index.js') );
+    this.log(this.destinationRoot())
+    this.log(this.destinationPath(this.classname + '.js'))
+    this.log(this.sourceRoot())
+    this.log(this.templatePath('index.js'))
   },
 
   // The name `constructor` is important here
```

Comment:
Details added: https://github.com/igorlima/generator-js-type/commit/26a3aa8fc4cf6830e572bd5c6dfbd688a09d31b6

@sashanka let me know if it's good enough. thanks